<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TaskManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:white;
	border-radius:1px;
	border:1px solid rgb(130, 130, 130);
}

QTabBar::tab{
	color:black;
	background: transparent;
	border: none;
	padding-left: 15px;
	padding-right: 15px;
	padding-top: 3px;
	padding-bottom: 3px;
}

QTabBar::tab:selected {
	border-bottom : 3px solid rgb(51, 136, 255);
	padding-bottom: 3px;
}

QTabBar::tab:hover:!selected{
	color:rgb(190, 190, 190);
	padding-bottom: 3px;
}

QTabWidget::pane{
	border:none;
}

QTreeView::item:selected{
	background-color: rgb(205,232,255);
	color:black;
}

QTreeView::item:hover:!selected {
    background-color: rgb(229, 243, 255);
} 

QTreeView::item{
	min-height:30px;
}

QTreeView::branch:selected{
	background:rgb(205,232,255);
}

QTreeView::branch:hover:!selected{
	background:rgb(229, 243, 255);
}

QHeaderView::section {
	min-height:50px;
	border:none;
	border-right:1px solid qlineargradient(spread:pad, x1:0.484, y1:0.994318, x2:0.489, y2:0.244, stop:0 rgba(165, 165, 165, 255), stop:1 rgba(255, 255, 255, 255));
	border-bottom:1px solid rgb(189, 189, 189);
	background-color:white;
}

QHeaderView::section:hover{
	background-color:rgb(229, 243, 255);
}


QPushButton{
 	background-color:rgb(51,133,255);
	border:none;
	color:rgb(255,255,255);
	font: 16pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color:rgb(49,126,243);
	border:none
}

QPushButton:pressed{
	background-color:rgb(48,117,220);
	border:none}

QPushButton#minimizeButton{
	background:transparent;
	border:none;
	color:rgb(255,255,255);
	font: 20pt &quot;Segoe UI&quot;;
}
QPushButton#minimizeButton:hover{
	background-color:rgba(255,255,255,150);
}
QPushButton#minimizeButton:pressed{
	background-color:rgba(255,255,255,120);
}

QPushButton#closeButton{
	background:transparent;
	border:none;
	color:rgb(255,255,255);
	font: 20pt &quot;Segoe UI&quot;;
}
QPushButton#closeButton:hover{
	background-color:rgba(255,255,255,150);
}
QPushButton#closeButton:pressed{
	background-color:rgba(255,255,255,120);
}

QLabel#titleLabel{
	font:16pt white &quot;Segoe UI&quot; ;
}

QWidget#titleWidget{
	background-color:rgb(0, 118, 255);
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Task Manager</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minimizeButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>â•³ </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="centralTabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="processTab">
       <attribute name="title">
        <string>Processes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="processView"/>
        </item>
        <item>
         <widget class="QWidget" name="killProcessWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>480</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QPushButton" name="killProcessButton">
             <property name="text">
              <string>Kill</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="performanceTab">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="usageOptionList">
          <item>
           <property name="text">
            <string>CPU</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Memory</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="couPage">
           <widget class="UsagePlot" name="cpuUsagePLot" native="true">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>211</width>
              <height>91</height>
             </rect>
            </property>
           </widget>
           <zorder>usageOptionList</zorder>
           <zorder>cpuUsagePLot</zorder>
          </widget>
          <widget class="QWidget" name="memoryPage">
           <widget class="UsagePlot" name="memoryUsagePlot" native="true">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>70</y>
              <width>231</width>
              <height>121</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="systemTab">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UsagePlot</class>
   <extends>QWidget</extends>
   <header location="global">UsagePlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
